includes/core/classes/class-export.php

```diff
@@ -96,6 +96,18 @@ public function prepare( WP_Post $post ): void {
 		}
 	}
 
+	/**
+	 * Checks if the currently exported post is of type 'gatherpress_event'.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param  WP_Post $post Current meta key.
+	 * @return bool          True, when the currently exported post is of type 'gatherpress_event', false otherwise.
+	 */
+	protected function validate( WP_Post $post ): bool {
+		return ( Event::POST_TYPE === $post->post_type );
+	}
```

Comment:
Let's add this to the new `Validate` class. `Validate::event_post_type( $post )`

https://github.com/GatherPress/gatherpress/blob/develop/includes/core/classes/class-validate.php

Reply:
never mind, we can use `Validate::event_post_id` here.